app.js

```diff
@@ -0,0 +1,41 @@
+//puts Javascript in 'expert mode'
+"use strict";
+
+//immediately-invoked function express (IIFE). Keeps the app from having many global variables
+(function(){
+//declaring the components. always begins with "angular"
+  angular
```

Comment:
I really like that you take the time to comment your code! I would suggest trying to use just one or two lines for each function just for readability purposes.